docs/guides/integration-prebid.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: Prebid Integration
-sidebar_label: Prebid
-pagination_label: Prebid Integration
+id: integration-prebid-advanced
+title: Prebid.js Advanced Integration
+sidebar_label: Prebid.js Advanced Integration
+pagination_label: Prebid Advanced Integration
```

Comment:
Shall we rename this page to integration-prebid-advanced too ?

also change in the first paragraph from:

_This guide is for publishers who want to integrate with UID2 and generate UID2 tokens (advertising tokens) to be passed by Prebid in the RTB bid stream._

to this:

_This guide is for publishers who want to integrate with UID2 by explicitly generating and/or refreshing UID2 tokens (advertising tokens) on server side and manually passing it into Prebid to then be sent into the RTB bid stream.

There is a simpler way to do Prebid Integration which does not require manual token generation, please refer to this (add link to the new integration page)._